server/registry/resources.py

```diff
@@ -41,6 +41,11 @@ def get_dogs(self):
 	def add_dog(self):
 		self.all_dogs.append(self)
 
+    def get_dog(id1, self):

```

Comment:
why is this called id1?

Reply:
I wasnt sure if there will be any errors cause an id has already been defined so just called it id1